content/src/content/jcr_root/apps/core/wcm/components/modal/v1/modal/modal.html

```diff
@@ -0,0 +1,33 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe Systems Incorporated
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+
+<div class="cmp-modal" data-sly-use.modal="com.adobe.cq.wcm.core.components.models.Modal"
+     data-sly-use.templates="core/wcm/components/commons/v1/templates.html"
+     role="dialog">
+    <sly data-sly-test="${wcmmode.edit}">${modal.title || 'Dialog' @ i18n} - ${modal.id}</sly>
+    <sly data-sly-test="${!wcmmode.edit}">
+        <sly data-sly-test.cfPathWithExt="${modal.fragmentPath @ extension='cfm.content.html'}"></sly>
+        <sly data-sly-test.xfPathWithExt="${modal.fragmentPath @ extension='html'}"></sly>
+        <div class="cmp-modal__view"
+             id="${modal.id}"
+             data-modal-show="${properties.open}"
+             tabindex="-1"
+             role="dialog"
+             data-content-url="${properties.fragmentType == 'cf' ? cfPathWithExt : xfPathWithExt}">
+        </div>
+    </sly>
+</div>
+<sly data-sly-call="${templates.placeholder @ isEmpty = wcmmode.edit, classAppend, emptyTextAppend = appendText}"></sly>

```

Comment:
Not sure what exactly you're trying to achieve here as the class and text appends are not defined, instead we have some output (from previous tags) even if `isEmpty` is true.

Reply:
Removed.